connectors/rsk.go

```diff
@@ -391,6 +394,15 @@ func (rsk *RSK) RegisterPegIn(opt *bind.TransactOpts, q bindings.LiquidityBridge
 	return rsk.lbc.RegisterPegIn(opt, q, signature, btcRawTrx, partialMerkleTree, height)
 }
 
+func (rsk *RSK) RegisterPegInWithoutTx(q bindings.LiquidityBridgeContractQuote, signature []byte, tx []byte, pmt []byte, height *big.Int) error {
```

Comment:
Make parameter names the same as in RegisterPegIn for consistency (or the other way around).

Reply:
done. I like this way more so I changed all to be the same as here.